src/client/containers/Profile/index.js

```diff
@@ -1,5 +1,166 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
+import Loader from '../../components/Loader/Loader.component';
+import { useAuthentication } from '../../hooks/useAuthentication';
 
 export default function Profile() {
-  return <div>This is your private profile</div>;
+  const [profileData, setProfileData] = useState(null);
+  const [profileIsLoading, setProfileLoading] = useState(true);
+  const [showForm, setShowForm] = useState(false);
+  const [newFullName, setFullName] = useState(null);
+  const [newMobile, setMobile] = useState(null);
+  const [newZipcode, setZipcode] = useState(null);
+  const [newCity, setCity] = useState(null);
+  const [newCountry, setCountry] = useState(null);
+  const [newAddress, setAddress] = useState(null);
+  const { user, isLoading } = useAuthentication();
+
+  function updateUserProfile(event) {
+    event.preventDefault();
+
+    console.log({
+      newFullName,
+      newMobile,
+      newZipcode,
+      newCity,
+      newCountry,
+      newAddress,
+    });
+    fetch(`/api/users/${user.uid}`, {
+      method: 'PATCH',
```

Comment:
I was not able to update my profile, I think it is because in our db we don't have user.uid.
#How to test information in the PR would be useful in this case.

Reply:
Sorry about that.
I have updated the description. Thanks for noticing.
As I mentioned in the description. we changed DB schema **user.id(integer) to user.uid(string)**